.travis.yml

```diff
@@ -1,21 +1,38 @@
-sudo: false
+if: tag IS present OR type = pull_request OR (branch = master AND type = push)   # we only CI the master, tags and PRs
+
 language: python
 # cache package wheels (1 cache per python version)
 cache: pip
-python:
-  - "3.4"
+dist: xenial
```

Comment:
While *I* know why xenial is needed for some of these runs, it would be nice to leave a short comment for the future.